### PR TITLE
Adding ICARUS NuMI Oscillation Fcls

### PR DESCRIPTION
**Pull request by jzennamo**
These fcls should enable the generation of a full osc and intrinsic nue-only samples for NuMI in the ICARUS detector. 

This should address https://github.com/SBNSoftware/icaruscode/issues/235

### CODE REVIEW

File: fcl/gen/numi/prodcorsika_genie_protononly_icarus_numi_fullosc_volDetEnclosure_tpc.fcl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include "prodcorsika_genie_protononly_icarus_numi_volDetEnclosure_tpc.fcl"
```
Comment (PetrilloAtWork) [inline fcl/gen/numi/prodcorsika_genie_protononly_icarus_numi_fullosc_volDetEnclosure_tpc.fcl:1]: I suggest to add a comment to describe in words what "full oscillation" means, like:
```suggestion
#
# Purpose: NuMI neutrino interactions (after "full" oscillation) and cosmic ray generation in ICARUS.
#
# Full oscillation: all muon neutrinos and antineutrinos are turned into electron ones, and vice versa.
# Generation is limited to an envelope volume of all TPC active volumes (plus a small margin).
#

#include "prodcorsika_genie_protononly_icarus_numi_volDetEnclosure_tpc.fcl"
```
And you can also add a `# Author: Joseph Zennamo (jaz8600@fnal.gov)` to the mix, for blaming purposes.

**Review by PetrilloAtWork — APPROVED** [review]
Suggesting the addition of an extended comment on the configuration that is less obvious.
But it is in general good.

File: fcl/gen/numi/prodcorsika_genie_protononly_icarus_numi_fullosc_volDetEnclosure_tpc.fcl (hunk 1 above)
Comment (jzennamo) [reply fcl/gen/numi/prodcorsika_genie_protononly_icarus_numi_fullosc_volDetEnclosure_tpc.fcl:11]: Done!

**Review by PetrilloAtWork — APPROVED** [review]
Approved without prejudice.

**Pull request merged**